src/averageAges.js

```diff
@@ -55,7 +94,33 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const peopleNew = [];
+  let bornChildren = 0;
+  let peopleNew2 = {};
```

Comment:
Variables names are not descriptive ennough. Consider providing more valuable names